lambda-functions/notification-template-formatter-service/src/main/java/com/paladincloud/FetchNotificationSettings.java

```diff
@@ -154,6 +157,31 @@ public String handleRequest(Map<String,Object> event, Context context)
         return response;
     }
 
+    private String buildNotificationPlainTextMail(String mailBody, Map<String, Object> messageContentMap, String source) {
+            mailBody=mailBody.replace("${source}",source);
+            mailBody=mailBody.replace("${message}",messageContentMap.get("message").toString());
+            StringBuilder buf = new StringBuilder();
+        for (Map.Entry<String, Object> permission:messageContentMap.entrySet()) {
+            if(permission.getKey().startsWith("permission"))
+            buf.append("<tr><td class=\"rowkey\">")
+                    .append(makeItReadable(permission.getKey()))
+                    .append("</td><td>")
+                    .append(permission.getValue())
+                    .append("</td></tr>");
+        }
+        mailBody=mailBody.replace("${permissionIssues}",buf.toString());
+        return mailBody;
+    }
+
+    private String makeItReadable(String key) {
+        key= key.replace("permission","Permission ");
+        key= key.replace("Issue","issue ");
+        key= key.replace("For","for ");
+        key= key.replace("In"," in ");
+        key= key.replace("Account","account ");
+        return key;
+    }
```

Comment:
Oh... Why we need this? 
Do we have a problem with messages? Do we produce them or we get them from somewhere?
If we produce them - why don't just solve problem itself instead of making this patch?

Reply:
When we send request in JSON format , there is no space in the key which results into unreadable format

![image](https://github.com/PaladinCloud/CE/assets/103987424/cebeb514-1962-47b8-90d2-031308de120d)